providers/chargebee/handlers.go

```diff
@@ -147,3 +150,79 @@ func (c *Connector) parseReadResponse(
 		params.Fields,
 	)
 }
+
+func (c *Connector) buildWriteRequest(ctx context.Context, params common.WriteParams) (*http.Request, error) {
+	fullObjectName := objectNameWrite.Get(params.ObjectName)
+
+	url, err := urlbuilder.New(c.ProviderInfo().BaseURL, apiVersion, fullObjectName)
+	if err != nil {
+		return nil, err
+	}
+
+	if params.RecordId != "" {
+		url.AddPath(params.RecordId)
+	}
+
+	// Convert RecordData to form values
+	formData := make(neturl.Values)
+
+	if recordMap, ok := params.RecordData.(map[string]any); ok {
+		for key, value := range recordMap {
+			if str, ok := value.(string); ok {
+				formData.Set(key, str)
+			} else if value != nil {
+				formData.Set(key, fmt.Sprintf("%v", value))
```

Comment:
Is this ever going to succeed as a write? Or is it only applicable for bool/int but not an array?